uber/config.py

```diff
@@ -120,6 +120,18 @@ def PAGE_PATH(self):
     def PAGE(self):
         return cherrypy.request.path_info.split('/')[-1]
 
+    @property
+    def SUPPORTER_COUNT(self):
```

Comment:
This function is almost identical to the BADGES_SOLD property. If we do more _COUNT properties they will probably follow the same format, so this is definitely a candidate for future refactoring.